src/services/housing.js

```diff
@@ -122,8 +132,8 @@ const getApartmentApplication = async (applicationID) => {
 
 export default {
   checkHousingStaff,
-  getApplicationID,
+  getCurrentApplicationID,
   saveApartmentApplication,
-  changeApplicationEditor,
+  changeApartmentAppEditor,
   getApartmentApplication,
 };
```

Comment:
Besides that the naming convention changes and comments in this section look good.